docs/installing.rst

```diff
@@ -154,7 +154,7 @@ The following virtualization providers are suggested for the respective operatin
 
 For some operating systems, you might need to install additional dependencies before you install the Vagrant plugins.
 
-* Fedora 24
+* Fedora 22/23/24
```

Comment:
@praveenkumar It's there in other places too. So I would probably let it be here and clean it later.